### PR TITLE
Update fault directory error msg

### PR DESCRIPTION
**Pull request by Lumsterling**
## Linked Issues:
| EPIC ⚡ | N/A |
| :----------- | :----------- |
| Jira Ticket 🎟️ | N/A |
| Github :octocat:  | # |

## Summary of Changes
* Very small change, simply updated a variable name as well as the error message when a custom fault file fails to open. It now displays the path attempted to reach the file rather than just the file name, making it easier to pinpoint possible problems in the code.

## Test
* roslaunch ow europa_terminator_workspace.launch
* In the RQT window, navigate to the Dynamic Reconfigure tab, then to the faults section. Scroll down to the custom faults section.
* Input an invalid file name to the `custom_fault_profile` text box, then check the `activate_custom_fault` box and observe the terminal.
* The terminal should output an error message that includes the full path to the file that was attempted to open.

### CODE REVIEW

**Review by anthonykoutroulis — COMMENTED** [review]
Looks good. Printing the full path additionally reveals working-directory/sourced-directory mismatch :+1:

**Comment by Lumsterling** [comment]
@anthonykoutroulis Unsure if this comment is approving the changes or not; normally the reviewer has the option to approve the merge or request further changes. Though because this is such a small change, if there's no response by 5pm I'll go ahead and merge.

**Comment by kmdalal** [comment]
@Lumsterling, one should wait until the formal approval of all reviewers.  There is no urgency to merge this.

**Comment by Lumsterling** [comment]
Makes sense, thanks for the clarification!

**Review by anthonykoutroulis — APPROVED** [review]

**Comment by anthonykoutroulis** [comment]
@Lumsterling I was just pointing out that the patch help me catch my original issue, definitely approved.

**Pull request merged**
